final release/model_sklearn.py

```diff
@@ -1,14 +1,37 @@
+"""Creates various Sklearn models"""
+
 import keras 
 import numpy as np
-from sklearn.ensemble import *
-from sklearn.linear_model import *
-from sklearn.svm import *
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.linear_model import LogisticRegression
+from sklearn.svm import LinearSVC
 
-def model_sklearn(train_x, train_y, method, params):
+def RFT(train_x, train_y, params):
+    """Instantiates random forest tree."""
+    
     train_x = train_x.reshape((train_x.shape[0], -1))
     
-    model = method(**params)
+    model = RandomForestClassifier(**params)
     model = model.fit(train_x, train_y)
     
     return model  
 
+def logistic_reg(train_x, train_y, params):
```

Comment:
You could merge all of these functions into a generic function and then make specific versions.

```python
def from_sklearn(Model):
    def f(train_x, train_y, params):
        ...
        model = Model(**params)
        ...
    
    return f

logistic_regression = from_sklearn(LogisticRegression)
random_forests = from_sklearn(RandomForestClassifier)
```

Reply:
can you clarify? How is this different to what I had in the first place?

Reply:
It's very similar to the original except that the API is different.

Reply:
I assume you mean

Reply:
def from_sklearn(Model, train_x, train_y, params):
    def f(train_x, train_y, params):
        ...
        model = Model(**params)
        ...
    
    return f